### DISCUSSION TITLE
CSV Export Not Capturing Updated Data

### DISCUSSION THREAD

**Discussion by DFWDesignGuy** [code4recovery/12-step-meeting-list#1486] [General]
After we go in and make meeting updates in the plugin, we export the CSV and that export does not reflect the changes we just made.  How can we export the most current version of the database?

**Comment by ginslo** [comment]
Hi @DFWDesignGuy. Your exports should reflect the most recent changes. I suspect there may be some data caching in your case. Are you able to share some details about your infrastructure? In particular, your hosting provider and DNS provider? Also can you share the URL to the website where you are seeing this condition? That will help us to try to figure this out. Thanks.

**Reply by DFWDesignGuy** [reply]
My apologies... we determined it is user error.  The export is working correctly.  Thank you for your quick response.

**Comment by DFWDesignGuy** [comment]
Gordon... Good news, as per my previous reply, the export is working. To take this one step further, the reason the export was being done was to import into an Access database to generate a PDF version of the directory. While we are aware the plugin CAN generate a PDF today, compared to the PDF the client has been generating (see attached), it is not as detailed. 

Our question is, would it be possible to through Code4Recovery to be able to generate the PDF is the format like what we have via the Access database so it can be done dynamically without having to go through the multi-step process to get it into Access.

Please advise and thank you!
Rick

[08092024.pdf](https://github.com/user-attachments/files/16566892/08092024.pdf)

**Reply by ginslo** [reply]
Ok, good to know the export is working for you.

Your directory looks great. And I am familiar from firsthand experience with a process similar to what you are describing. The PDF generator designed by Code4Recovery for use with the 12 Step Meeting Guide is great, but is not able to generate a report in your format. If you have not tried it, simply go to [https://pdf.code4recovery.org/ ](https://pdf.code4recovery.org/ )and enter the feed to your meeting data.
